outrank/algorithms/feature_ranking/ranking_cov_alignment.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import annotations
+
+import numpy as np
+
+np.random.seed(123)
+max_size = 10**6
+
+
+def max_pair_coverage(array1: np.array, array2: np.array) -> float:
+    def hash_pair(el1, el2):
+        return el1 * 17 - el2
```

Comment:
this is around 100% faster to `hash(el1) someOp hash(el2)` and is fine for this use case col wise